src/algorithms/otsu.jl

```diff
@@ -1,6 +1,6 @@
 @doc raw"""
 ```
-t = find_threshold(Otsu(), histogram, edges)
+t = Otsu(histogram, edges)
 ```
```

Comment:
The signature is actually not `Otsu(histogram, edges)` but `(::Otsu)(histogram, edges)`. How about:

```suggestion
    t = find_threshold(histogram, edges, Otsu())
```

Reply:
Ah yes, of course! I also forgot to update the example in that docstring.